integration_tests/cork_test.go

```diff
@@ -154,27 +147,87 @@ func (s *IntegrationTestSuite) TestCork() {
 			return true
 		}, 105*time.Second, 1*time.Second, "new vote period never seen")
 
-		s.T().Logf("checking for updated tick ranges in cellar")
+		s.T().Logf("waiting for gravity to submit call to cellar")
 		s.Require().Eventuallyf(func() bool {
-			actualTickRange, err := s.getFirstTickRange()
-			if err != nil {
-				s.T().Logf("got error %e querying ticks", err)
-				return false
+			s.T().Log("querying gravity logic call events...")
+			ethClient, err := ethclient.Dial(fmt.Sprintf("http://%s", s.ethResource.GetHostPort("8545/tcp")))
+			s.Require().NoError(err)
+
+			query := ethereum.FilterQuery{
+				FromBlock: nil,
+				ToBlock:   nil,
+				Addresses: []common.Address{
+					gravityContract,
+				},
 			}
-			if expectedTickRange.Upper != actualTickRange.Upper {
-				s.T().Logf("wrong upper %s", actualTickRange.String())
-				return false
+			logs, err := ethClient.FilterLogs(context.Background(), query)
+			s.Require().NoError(err)
+			s.T().Logf("got %v logs", len(logs))
+
+			// For non-anonymous events, the first log topic is a keccak256 hash of the
+			// event signature.
+			eventSignature := []byte("LogicCallEvent(bytes32,uint256,bytes,uint256)")
+			signatureTopic := crypto.Keccak256Hash(eventSignature).Hex()
+			s.T().Logf("expected signature topic: %s...", signatureTopic[:10])
+
+			for _, vLog := range logs {
+				var topics [4]string
+				for i := range vLog.Topics {
+					topics[i] = vLog.Topics[i].Hex()
+				}
+
+				s.T().Logf("found signature topic: %s...", topics[0][:10])
+				if topics[0] == signatureTopic {
+					s.T().Logf("gravity logic call event found!")
```

Comment:
Should we be verifying the contents of the event match the call we're looking for? We're just verifying the presence of one on that topic.

Reply:
Probably a good idea... It is the only `submitLogicCall` call occurring in the test but could cause weird false positives in the future if that changes.